src/Core/Collections/CustomDropdown.jsx

```diff
@@ -0,0 +1,26 @@
+import React, { Component } from "react";
+
+import "./_custom-dropdown.scss";
+
+export default class CustomDropdown extends Component {
+    onCollectionChange = (event) => {
+        this.props.onChange(event.target.value)
+    }
+    render() {
+        const {optionArray} = this.props;
+        return (
+            <div className={"custom-dropdown-wrapper"}>
+                <div className={"shadow"} />
```

Comment:
```suggestion
                <div className={"shadow"}/>
```

And put an empty line after this line.